modules/data.atmosphere/R/download.CRUNCEP_Global.R

```diff
@@ -24,9 +24,9 @@
 ##' @author James Simkins, Mike Dietze, Alexey Shiklomanov
 download.CRUNCEP <- function(outfolder, start_date, end_date, site_id, lat.in, lon.in,
                              overwrite = FALSE, verbose = FALSE, maxErrors = 10, sleep = 2,
-                             method = "opendap", ...) {
+                             method = "ncss", ...) {
 
-  if (is.null(method)) method <- "opendap"
+  if (is.null(method)) method <- "ncss"
```

Comment:
Given the default value in the function signature, couldn't this line be removed? Or do we need to support calls that explicitly set method=NULL for some reason?

Reply:
I'm not really the developer of this function but my best guess is that it needs to be able to support that because `convert.input.R` function creates a generic set of argument that is used for all the `*met.product*.download.R` functions.

Reply:
That's fair. Probably not worth the upstream effort to change that, then.